### PR TITLE
Test linguistics-tokens summary transform.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

This tests the changes in vespa-engine/vespa#28943

### CODE REVIEW

File: tests/search/linguistics_tokens/linguistics_tokens.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'indexed_search_test'
+
+class DynTeaserArrayTest < IndexedSearchTest
```
Comment (geirst) [inline tests/search/linguistics_tokens/linguistics_tokens.rb:4]: Please rename to LinguisticsTokensTest (or similar).

File: tests/search/linguistics_tokens/linguistics_tokens.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  def feed_doc(idsuffix, doc_template)
+    doc = Document.new("test", "id:test:test::#{idsuffix}").
+      add_field("stext", doc_template[:stext]).
+      add_field("atext", doc_template[:atext]).
+      add_field("wtext", doc_template[:wtext])
+    vespa.document_api_v1.put(doc)
+  end
+
+  def test_lingustics_tokens
+    set_description("Test linguistics tokens dfw")
```
Comment (geirst) [inline tests/search/linguistics_tokens/linguistics_tokens.rb:19]:
```suggestion
    set_description("Test linguistics-tokens summary transform for inspecting indexed tokens")
```

File: tests/search/linguistics_tokens/test.sd [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    field atext type array<string> {
+      indexing: index | summary
+    }
+    field wtext type weightedset<string> {
+      indexing: index | summary
+    }
+  }
+  document-summary tokens {
+    summary stext type string { }
```
Comment (geirst) [inline tests/search/linguistics_tokens/test.sd:15]: Summary fields 'stext', 'atext' and 'wtext' do not seem to be used by the test, consider removing.

**Review by geirst — DISMISSED** [review]
👍

File: tests/search/linguistics_tokens/linguistics_tokens.rb (hunk 1 above)
Comment (toregge) [reply tests/search/linguistics_tokens/linguistics_tokens.rb]: Renamed.

File: tests/search/linguistics_tokens/linguistics_tokens.rb (hunk 2 above)
Comment (toregge) [reply tests/search/linguistics_tokens/linguistics_tokens.rb]: Description changed.

File: tests/search/linguistics_tokens/test.sd (hunk 3 above)
Comment (toregge) [reply tests/search/linguistics_tokens/test.sd]: Removed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**